webapp/src/components/CreateIcon.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+
+export default function CreateIcon({
+  iconVariant,
+  bgColour,
+  bgIcon,
+  customStyles,
+  bgIconCustomStyles,
+}) {
+  let iconStyle = "";
+  let iconStyleInner = "";
+
+  const getStyles = () => {
+    if (iconVariant === "type1") {
```

Comment:
This variant type is considered as an `enum`. You can have this under `constants.js` and access from there. This will improve the consistency